frontend/src/lib/components/PropertyFilters/components/taxonomicPropertyFilterLogic.test.ts

```diff
@@ -14,6 +15,7 @@ describe('the taxonomic property filter', () => {
 
     beforeEach(() => {
         initKeaTests()
+        columnConfiguratorLogic({ selectedColumns: [] }).mount()
```

Comment:
Hmm... Will this test fail without it? If so, it's coupled in a strange way. Something to improve?

Reply:
it's too yucky, I'll remove